mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -1,9 +1,22 @@
 CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_17884 (Ljava/util/Map;Lnet/minecraft/class_2487;Lnet/minecraft/class_1887;)V
 		ARG 2 enchantment
-	METHOD method_22445 getEnchantments (Lnet/minecraft/class_2499;)Ljava/util/Map;
+	METHOD method_22445 fromTag (Lnet/minecraft/class_2499;)Ljava/util/Map;
+		COMMENT Loads enchantments from a list named binary tag.
```

Comment:
```suggestion
		COMMENT Loads enchantments from a NBT list.
```
I initially read "a list named binary tag" as "a list named 'binary tag'"

Reply:
I think 'an NBT list' would be more correct

Reply:
I hesitated on which to take. "an NBT list" does sound better if you say it as an acronym.

Reply:
Done